.storybook/preview.js

```diff
@@ -1,4 +1,20 @@
 
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import GlobalStyle from '../src/components/GlobalStyle/GlobalStyle'
+
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
-}
\ No newline at end of file
+  viewport: {
+    viewports: INITIAL_VIEWPORTS,
+    defaultViewport: 'iphone6',
+  },
+}
+
+export const decorators =[
+  Story => (
+    <>
+      <GlobalStyle />
+      <Story />
+    </>
+  ),
+]
```

Comment:
Please configure your editor to add an empty line on save to avoid possible errors in some tools.